src/FEValues/GeometryMapping.jl

```diff
@@ -35,7 +35,7 @@ struct GeometryMapping{DiffOrder, IP, M_t, dMdξ_t, d2Mdξ2_t}
     M::M_t             # ::AbstractMatrix{<:Number}     Values of geometric shape functions
     dMdξ::dMdξ_t       # ::AbstractMatrix{<:Vec}        Gradients of geometric shape functions in ref-domain
     d2Mdξ2::d2Mdξ2_t   # ::AbstractMatrix{<:Tensor{2}}  Hessians of geometric shape functions in ref-domain
-                       # ::Nothing                      When not required
+                       
```

Comment:
Somewhere apart from the code it would be nice to mention that these can also be of type `Nothing`, I just found the `Union{Nothing, AbstractMatrix{...}}` notation to verbose above. 
```suggestion
                       # ::Nothing (dMdξ or d2Mdξ2 if not required)
```